Makefile

```diff
@@ -90,30 +97,31 @@ precommit: cover gofmt lint test
 $(COMPILER_BIN) $(RUNNER_BIN) $(TOOL_BINS): build/bin/%: tools/%
 	@mkdir -p build/bin
 	@cp -f $< $@
+	@sed -i '' '1s@/usr/bin/env python@$(PYTHON_BIN)@' $@
```

Comment:
Looks like `cp -f $< $@` is not done yet when sed try to read content of the file.
Not sure it is because of my xfs file system.